.github/workflows/ci.yml

```diff
@@ -585,10 +585,10 @@ jobs:
         run: env
         if: ${{ always() }}
 
-  macos-native:
-    name: "x86_64: macOS Monterey"
+  x86_64-macos-native:
+    name: "x86_64: macOS Monterey, Valgrind"
```

Comment:
Did you remove the "Valgrind" intentionally?

Reply:
Do you mean in the https://github.com/bitcoin-core/secp256k1/pull/1412?

Reply:
No, I mean you have removed the substring ", Valgrind" from the job name. :D

Reply:
Sorry, I don't follow... The diff _appends_ the substring ", Valgrind" to the job name, no?

Reply:
Lol sure, sorry, nevermind...